meteor/client/ui/ClockView/OverlayScreen.tsx

```diff
@@ -23,94 +27,112 @@ interface RundownOverviewState {}
 /**
  * This component renders a Countdown screen for a given playlist
  */
-export const OverlayScreen = withTranslation()(
-	withTracker<RundownOverviewProps & WithTranslation, RundownOverviewState, RundownOverviewTrackedProps>(
-		getPresenterScreenReactive
-	)(
-		withTiming<RundownOverviewProps & RundownOverviewTrackedProps & WithTranslation, RundownOverviewState>()(
-			class OverlayScreen extends PresenterScreenBase {
-				protected bodyClassList: string[] = ['transparent']
+export const OverlayScreen = withTracker<RundownOverviewProps, RundownOverviewState, RundownOverviewTrackedProps>(
+	getPresenterScreenReactive
+)(
+	withTiming<RundownOverviewProps & RundownOverviewTrackedProps, RundownOverviewState>()(function OverlayScreen(
+		props: Readonly<WithTiming<RundownOverviewProps & RundownOverviewTrackedProps>>
```

Comment:
rename `RundownOverviewProps` to `IProps` perhaps?

Reply:
any reason for that change?
I've done a bunch of renames in the other direction elsewhere as `IProps` is really unclear on what it is (especially when there are multiple components in a file)

Reply:
renamed to OverlayScreenProps